### PR TITLE
Completed javascript-timers exercise.

### PR DESCRIPTION
**Pull request by RossBrewster**
https://github.com/RossBrewster/c-0823-code-solutions/assets/124334539/b4036611-4538-4896-a908-c841a6852de6

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+function displayNextMessage() {
+  if ($countdownDisplay.textContent === '4') {
+    $countdownDisplay.textContent = '3';
+  } else if ($countdownDisplay.textContent === '3') {
+    $countdownDisplay.textContent = '2';
+  } else if ($countdownDisplay.textContent === '2') {
+    $countdownDisplay.textContent = '1';
+  } else if ($countdownDisplay.textContent === '1')
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+}
```
Comment (shawnkost) [inline javascript-timers/interval.js]: You should be using `clearInterval` inside of here

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const $countdownDisplay = document.querySelector('.countdown-display');
+// const $message = $countdownDisplay.textContent;
+// let intervalId;
```
Comment (shawnkost) [inline javascript-timers/interval.js]: Delete all old code

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Ross!

**Comment by RossBrewster** [comment]
Not sure I included the clearInterval method properly.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**